label_studio/tests/data_manager/api_tasks.tavern.yml

```diff
@@ -4,7 +4,7 @@ strict: false
 marks:
   - usefixtures:
       - django_live_url
-  - skipif: "'default' not in '{tavern.env_vars.DJANGO_DB}'"
+  - skipif: "'default' not in '{tavern.env_vars.DJANGO_DB}' or 'Darwin' in platform.system()"
```

Comment:
I'm not sure about it. 
LabelStudio support PostgreSQL and sqlite database backends. Idea of this skip was to check PostgreSQL related code only when we setup project to use it. 

If you don't want to run these tests on `Darwin` system, you should control it over environment variables on tests run. 

How we run tests in our CI: https://github.com/heartexlabs/label-studio/blob/develop/.github/workflows/tests.yml#L238

Place in the code, where we read this env variable: https://github.com/heartexlabs/label-studio/blob/develop/label_studio/core/settings/base.py#L101

Reply:
> If you don't want to run these tests on Darwin system, you should control it over environment variables on tests run.

I would like to run them, however,  this test fails on my system. Apple M1. I spent much time trying to resolve it. A updated test which passes on my system fails as part when run from ci on linux.

I agree it's better to not skip but since Linux is our target production architecture I thought it would not matter so much, there are tests we skip on Windows as well.